go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/Zbyteio/middleware-lib
+module github.com/sourabhmandal/testzbytemiddleware
```

Comment:
Why it is referring to personal repo?

Reply:
was testing the latest package so that it is installable. will revert back to origional